config/mapzen-js.yml

```diff
@@ -0,0 +1,13 @@
+site_name: Mapzen Javascript SDK
```

Comment:
Let's make it JavaScript, with upper S. 

Or perhaps it's just Mapzen.js (styled however) with the words JavaScript SDK in the text underneath?